devtools/format_yml.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+command -v shellcheck > /dev/null && shellcheck "$0"
```

Comment:
This is silly, but I prefer  `>/dev/null` (without a space) to indicate that the output goes "inside" the file/device. :-)

Reply:
We need a prettier for shell scripts :) Maybe we should give this plugin a try: https://www.npmjs.com/package/prettier-plugin-sh

Reply:
OK, I'll create the issue not to forget about it. :-)

Reply:
#652.